src/components/card/card-row/card-row.component.js

```diff
@@ -1,19 +1,37 @@
 import React from "react";
+import styledSystemPropTypes from "@styled-system/prop-types";
 import PropTypes from "prop-types";
 import StyledCardRow from "./card-row.style";
 import OptionsHelper from "../../../utils/helpers/options-helper/options-helper";
+import { filterStyledSystemPaddingProps } from "../../../style/utils";
 
-const CardRow = ({ children, spacing, ...props }) => {
+const paddingPropTypes = filterStyledSystemPaddingProps(
+  styledSystemPropTypes.space
+);
+
+const paddingSizes = {
+  small: 2,
+  medium: 3,
+  large: 4,
+};
+
+const CardRow = ({ children, spacing = "medium", ...props }) => {
   return (
-    <StyledCardRow data-element="card-row" spacing={spacing} {...props}>
+    <StyledCardRow
+      data-element="card-row"
+      py={paddingSizes[spacing]}
+      {...props}
+    >
       {children}
     </StyledCardRow>
   );
 };
 
 CardRow.propTypes = {
+  /** Styled system padding props */
+  ...paddingPropTypes,
   children: PropTypes.node.isRequired,
-  /** size of card for applying margin (small | medium | large) */
+  /** Predefined size of card for applying padding (small | medium | large). For more granular control these can be over-ridden by Padding props from styled-system (see table below). */
```

Comment:
Could we make it more clear that padding is set by the spacing prop on Card and setting a spacing prop in CardRow will do nothing. 
```suggestion
  /** Spacing prop is set in Card and defines the padding for the CardRow (the first CardRow has no padding by default) For more granular control of CardRow padding these can be over-ridden by Padding props from styled-system (see table below). */
```